settings/provisioner/virsh/virsh.spec

```diff
@@ -8,28 +8,39 @@ subparsers:
                   host:
                       type: str
                       help: Address/FQDN of the BM hypervisor
+                      required: yes
                   ssh-user:
                       type: str
                       help: User to SSH to the host with
+                      default: root
+                      required: yes
```

Comment:
I've added that to have complete picture regarding all the spec arguments. 
Additionally, the default value can be overridden by env or ini file. 'Required' in will raise en exception when provided value is empty.